adala/environments/kafka.py

```diff
@@ -26,19 +26,43 @@ class AsyncKafkaEnvironment(AsyncEnvironment):
         kafka_bootstrap_servers (Union[str, List[str]]): The Kafka bootstrap servers.
         kafka_input_topic (str): The Kafka input topic.
         kafka_output_topic (str): The Kafka output topic.
+        timeout_ms (int): The timeout for the Kafka consumer.
     """
 
-    kafka_bootstrap_servers: Union[str, List[str]]
-    kafka_input_topic: str
-    kafka_output_topic: str
+    # these are mandatory, but should be set by server
+    kafka_bootstrap_servers: Optional[Union[str, List[str]]] = None
+    kafka_input_topic: Optional[str] = None
+    kafka_output_topic: Optional[str] = None
+    timeout_ms: Optional[int] = None
+
+    # these are set in initialize()
+    consumer: Optional[AIOKafkaConsumer] = None
+    producer: Optional[AIOKafkaProducer] = None
 
     async def initialize(self):
-        # claim kafka topic from shared pool here?
-        pass
+        assert self.kafka_bootstrap_servers is not None, "missing initialization"
```

Comment:
If we do have to keep with this workflow, we should at least state which param is missing in these asserts

Reply:
done